doc/programming_guide/install/windows.rst

```diff
@@ -76,3 +76,34 @@ If you happen to be using pipenv, then the appropriate command is:
 
 ``python3 -m pipenv install arcade``
 
+Install Arcade under WSL
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need to use arcade on Ubuntu WSL, here are multiple guides depending on what you want.
+You can also explore running headless.
+
+TODO: ^^^ link to docs page, help pls
+
+Launching with NVIDIA GPUs
+--------------------------
```

Comment:
Is this section to go under the WSL one? Using dashes will place it higher in the hierarchy. See:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections